### DISCUSSION TITLE
Ultra wide display priority section of screen

### DISCUSSION THREAD

**Discussion by eastbay74** [rxhanson/Rectangle#391] [Ideas]
I'll use an ultra wide scrren and I would like to split screen i 3 sections like thirds but with a bigger center. Like half the screen in center and 1/4 on each side of the center window. Would be nice to be able to configure the with of the center window and use whats left to the two widows on each side of it.

**Comment by rxhanson** [comment]
Thanks for the idea. For this scenario, I don't plan on adding anything into Rectangle at this time (sorry!).

There are so many desired edge cases for window placement and dimensions, that I did build a custom shortcut builder for my paid superset of Rectangle, Hookshot (https://hookshot.app). Here's a post I wrote about how to use it: https://medium.com/ryan-hanson/custom-window-dimension-shortcuts-in-macos-e7df347dfd24

**Reply by eastbay74** [reply]
I found the solution, I'll use first fourth, center half and last fourth and configured shortcuts for them so now I'm pleesed.
